test-resources-hibernate-reactive/test-resources-hibernate-reactive-mssql/src/test/groovy/io/micronaut/testresources/hibernate/reactive/mssql/WithJdbcStartMSSQLDBTest.groovy

```diff
@@ -0,0 +1,37 @@
+package io.micronaut.testresources.hibernate.reactive.mssql
+
+import io.micronaut.test.extensions.spock.annotation.MicronautTest
+import io.micronaut.testresources.hibernate.reactive.core.Book
+import io.micronaut.testresources.hibernate.reactive.core.BookRepository
+import io.micronaut.testresources.testcontainers.AbstractTestContainersSpec
+import jakarta.inject.Inject
+
+import java.time.Duration
+import java.time.temporal.ChronoUnit
+
+@MicronautTest(transactional = false, environments = "jdbc")
```

Comment:
Hibernate Relative is not using anything JDBC, so this `environments` entry is a bit confusing.